### ISSUE TITLE
paper review 

### ISSUE THREAD

**Issue by spencerangusthomas** [adamjtaylor/tablet-classification#10]
Natalie said to post them here and there are no line numbers so I've tred to be as clear as I can by lisiting the section for the minor corrections 

'Major' - no real issues with the work, some comments on the description of it 

- [x] The use and definition of abbreviations are not consistent in the paper 
- [x] Reference to supplementary figures is incorrect in several cases, most of these are not consistent with the text. 
- [x] The cross validation and data split is not clear at all and missing how many samples you are using. - - [ ] The repeatability section is the first time you mention sample numbers with 54 scans from tablet A but nothing about the others. 
- [x] You conclusions state that you see high precision and recall but you have only presented Fmeasure so although high precision and recall is implied by high Fmeasure this is not shown here so I would consider rewording this. 

Minor 
- [x] Not sure on the title ‘complementary classification’ implies that there are multiple methods for classification that complement each other, i.e. ensembles, but you are only using single models here 

Minor 

Introduction 
- [x] API, HPLC, FTIR, NIR, IR, MDMA not defined  
- [x] Chen et al does not have a citation 
- [x] Not wanting to be ‘that reviewer’ but given the nature of the paper and your statements of applications of classification algorithms I am surprised that you have not referenced this paper https://ieeexplore.ieee.org/document/8285223 

Ambient section 
- [x] QA/QC not defined in 
- [x] PLS-DA is not defined 
- [x] REIMS not defined 

Raman section 
- [x] Matousek et al is statement has 2 papers lists so a bit ambiguous 
- [x] SVM and LDA not defined 
- [x] DA (in PLS-DA) not defined 
- [x] ‘pre-preprocessing ´- … data quantity and the pre-preprocessing performed [43]
- [x] ransson el al should is all in lower case 

experimental section
- [x] does MHRA need to be defined? 

Data analysis
- [x] Some R packages are cited other are not, this should be consistent all or none, same should be for matlab functions 
- [x] you don’t mentioned how many instances you have in total to in each class 

Classification 
- [x] ‘Spectra were collated into a 10-fold cross validation 1 with 10 repeats in a 4/1/5 (train/test/total) split.’ This isn’t really clear what you’re doing. Do you ‘validation 1 with 10 ‘ doesn’t read well. 
- [x] Also are you leaving half the data out train/test/total ? the number look like you’re splitting into 10 folds but only doing 5 fold x-validation  
- [x] ‘polymer peak sequences leading to overtting’ - > ‘polymer peak sequences that lead to overtting’ also this should have a reference or justification as covariate features do not (obviously) lead to over fitting 
- [x] What method of up sampling did you use? 
- [x] further model tuning on a single 4/1/5 validation split of the training data’ is this the other split missing from the 10 folds?

Acquisition of DESI-MS section
- [x] Presence – present 
- [x] Different peak sequences are observed -> Different peak sequences observed
- [x] Does PEG need to be defined? 
- [x] Table -> tablet 

- [x] Figure 1 the used of A_D for the tablets and subfigures is not ideal perhaps use lower case for the subfigures to differentiate them from the tables 

Relative spectral similarity 
- [x] ‘The low standard deviation of cosine similarity within these tablet types indicates the highly reproducible spectra achieved from these samples.’ This would be reflected in a higher cosine similarity too, its not as simple as this statement implies. 
- [x] There is no mention of figure 2 D 

Assessment of calcination algorithms
- [x] You should use the abbreviated names mentioned earlier, you also redefine LDA
- [x] ‘While not a limitation in a the relatively’ -> ‘While not a limitation in the relatively’

Test set classification 
- [x] Use abbreviations as previously defined 
r- [ ] eference to the total ion chromatogram (Figure S3a). -> figure 3a 
- [x] The trained SVM model was then used to predict the classification of each scan (Figure S3b). -> there is no fig S3b
- [x] ‘Figure 2C shows the shows’
- [x] No figure S3f do you mean S4f?
- [x] Missing ) in using LDA (Figure 2F
- [x] ‘correct for tablet types B and C and A’ - -> ‘A,B and C’ ?

Variable importance 
- [x] (2 ref) -> missing citation? I assume this is the case as there are no details to how feature importance is determined.

**Comment by adamjtaylor** [comment]
Title updated to "Classification of tablet formulations by desorption electrospray ionization mass spectrometry and transmission Raman spectroscopy"

**Comment by adamjtaylor** [comment]
> API, HPLC, FTIR, NIR, IR, MDMA not defined
Now defined

**Comment by adamjtaylor** [comment]
> Chen et al does not have a citation. 

Cited in previous sentence, but structure altered to be clearer
> In one of the early applications of DESI MS, Chen *et al.* demonstrated the use of DESI MS to profile tablets containing loratadine, folic acid, acetaminophen (paracetamol), aspirin, melatonin or caffeine [@doi:10.1021/ac050989d].

**Comment by adamjtaylor** [comment]
> Matousek et al is statement has 2 papers lists so a bit ambiguous

References split for clarity

**Comment by adamjtaylor** [comment]
These largely look to be corrected. Will merge PR #12 and we can do a final review before submission
